src/components/nav/components/NavBar.js

```diff
@@ -0,0 +1,39 @@
+import { MenuOutlined, } from "@mui/icons-material"
+import { AppBar, Avatar, Badge, Grid, IconButton, Toolbar, Typography } from "@mui/material"
+import NotificationsIcon from '@mui/icons-material/Notifications';
+
+import {useLocation} from 'react-router-dom'
+
+export const NavBar = ({ drawerWidth = 255 }) => {
+  const {pathname} = useLocation()
+  return (
+    <AppBar
+      style={{ backgroundColor: '#F7F8FC', boxShadow: 'none' }}
```

Comment:
Don't use the styles and colors in the same file as the component. The files must be in a different file.